docs/design/3_2_multirotor.tex

```diff
@@ -84,32 +81,25 @@ \subsubsection{Multirotor Configuration}
     \label{fig:rpas_6dof}
 \end{figure}
 
-\textbf{Hexacopter X: }
-The hexacopter has all the stability benefits of a quadcopter. In addition, the hexacopter configuration also 
-offers redundancy. Up to 2 motors can fail and the MRPAS could still land safely. Due to the added 
-propulsion systems (50\% more than the quadcopter configuration), hexacopter can generally lift more 
-payload than quadcopters. The downside is that hexacopter configuration builds are more expensive to build, 
-operate, and maintain. Upon a crash, hexacopter MRPAS' extended size and weight can cause more injuries 
-or collateral damage to nearby equipment.
+\textbf{Hexacopter X:}
+The hexacopter has all the stability benefits of a quadcopter as well as extra redundancy. The MRPAS could still land safely even up to dual motor failures. Since the hexacopter uses two more motors than the quadcopter configuration, naturally, the RPAS of this configuration is generally bigger and can lift more payload. Of course, with the added components, it is also much more expensive to build, operate, and maintain. Furthermore, hexacopter RPAS's extended size and mass can cause more damage upon a crash.
```

Comment:
```suggestion
The hexacopter has all the stability benefits of a quadcopter as well as extra redundancy. The MRPAS could still land safely even if it experienced up to two simultaneous motor failures. Since the hexacopter uses two more motors than the quadcopter configuration, naturally, the RPAS of this configuration is generally bigger and can lift more payload. Of course, with the added components, it is also much more expensive to build, operate, and maintain. Furthermore, hexacopter RPAS's extended size and mass can cause more damage upon a crash.
```